ethereum/spec/src/main/java/tech/pegasys/teku/spec/executionlayer/ExecutionLayerChannelStub.java

```diff
@@ -586,11 +585,8 @@ private void prepareTransitionBlocks(final UInt64 bellatrixActivationTime) {
       terminalBlockHash = configTerminalBlockHash;
     }
 
-    terminalBlockParent =
-        new PowBlock(TERMINAL_BLOCK_PARENT_HASH, Bytes32.ZERO, UInt256.ZERO, UInt64.ZERO);
-    terminalBlock =
-        new PowBlock(
-            terminalBlockHash, TERMINAL_BLOCK_PARENT_HASH, terminalTotalDifficulty, transitionTime);
+    terminalBlockParent = new PowBlock(TERMINAL_BLOCK_PARENT_HASH, Bytes32.ZERO, UInt64.ZERO);
```

Comment:
we can do some additional cleanup here in the stub, see `terminalBlockHashInTTDMode`